addons/placement/functions/fnc_handleObjectPlaced.sqf

```diff
@@ -37,6 +37,7 @@ _object allowDamage false;
     _object setPosASL _position;
     _object setVectorDirAndUp _dirAndUp;
     _object setVelocity [0, 0, 0];
+    ["", _object] call EFUNC(building_markers,handleObjectPlaced);
```

Comment:
I'd say let's invert this. Have `placement` trigger a local event for the when the `_object` has been "fully placed" and then `building_markers` can listen for the event and update accordingly. Also, enables any future functionality to take advantage of this as well.

And maybe consider moving inside the `CBA_fnc_execNextFrame`.